src/main/java/com/ashkan/moviesapi/entities/MovieRental.java

```diff
@@ -1,4 +1,4 @@
-package com.ashkan.moviesapi.entity;
+package com.ashkan.moviesapi.entities;
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
 
```

Comment:
This comment/question should be in line 18:
According to the requirements, do you think you can use an Enum here?